app/controllers/static_pages_controller.rb

```diff
@@ -1,4 +1,5 @@
 class StaticPagesController < ApplicationController
 	def index
+		@games = Game.all
```

Comment:
you should filter write a scope on the Game model so that it only returns games with white_uid nil or black_id nil

i'll give you a hint..

in the app/models/game.rb

implement the following method to return the proper scope.

def self.joinable
  # your code here
end

then in the controller you will call Game.joinable

then remove the if condition from the view...